doc/lightning-openchannel_update.7.md

```diff
@@ -0,0 +1,59 @@
+lightning-openchannel\_update -- Command to update a collab channel open
+========================================================================
+
+SYNOPSIS
+--------
+
+**openchannel_update** *id* *psbt*
```

Comment:
Maybe these APIs should use channelid (in which case, figure out how we know that?)

Reply:
Good idea, will update!